### DISCUSSION TITLE
What is your vision for SeaweedAdmin server ?

### DISCUSSION THREAD

**Discussion by nkosi23** [seaweedfs/seaweedfs#6085] [General]
Hello!

I've just noticed the announcement for the close-source SeaweedAdmin server in the release notes of the latest release, so I am curious about what is your vision for the SeaweedAdmin server, and on the long run, how you see the respective roles of the open source system and the close source tools.

Anyway, congratulations for shipping this. You've done a tremendous work building seaweedfs so it's a good to see that you have found a way to make it sustainable and profitable. As usual, as a user i just hope that the community will keep having access to what we are used to.

Thanks

**Comment by chrislusf** [comment]
Maybe you can tell me what an ideal storage solution should look like?
